Resources/translations/AdminLTEBundle.en.xliff

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
-    <file date="2017-01-12T20:00:00Z" source-language="en" target-language="en" datatype="plaintext" original="none">
+    <file date="2019-04-10T18:45:00Z" source-language="en" target-language="en" datatype="plaintext" original="AdminLTEBundle.en.xliff">
```

Comment:
Sorry for the misunderstanding, I just meant your file - so please revert this one ;-)

Reply:
i accidentally updated the wrong file, oops